.env.example

```diff
@@ -0,0 +1,2 @@
+SERVER_URL=https://server.com
+DEFAULT_COMPUTE_CONTEXT=SAS Studio compute context
```

Comment:
We shouldn't ever use this context - it should be SAS Job Execution

Reply:
- Made requesting and printing job PID optional
- Fixed the example